bugbug/models/performancebug.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+import xgboost
+from imblearn.over_sampling import BorderlineSMOTE
+from imblearn.pipeline import Pipeline as ImblearnPipeline
+from sklearn.compose import ColumnTransformer
+from sklearn.feature_extraction import DictVectorizer
+from sklearn.pipeline import Pipeline
+
+from bugbug import bug_features, bugzilla, feature_cleanup, utils
+from bugbug.model import BugModel
+from bugbug.models.bugtype import bug_to_types
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def _is_performance_bug(bug_data) -> bool:
+    types = bug_to_types(bug_data)
+    return "performance" in types
+
+
+class PerformanceBugModel(BugModel):
+    def __init__(self, lemmatization=False):
+        BugModel.__init__(self, lemmatization)
+
+        self.calculate_importance = False
+
+        feature_extractors = [
+            bug_features.HasSTR(),
+            bug_features.Keywords({"perf", "topperf", "main-thread-io"}),
+            bug_features.IsCoverityIssue(),
+            bug_features.HasCrashSignature(),
+            bug_features.HasURL(),
+            bug_features.HasW3CURL(),
+            bug_features.HasGithubURL(),
+            bug_features.BlockedBugsNumber(),
+            bug_features.Product(),
+            bug_features.HasRegressionRange(),
+            bug_features.BugReporter(),
```

Comment:
I do not think we should have `BugReporter` as a feature. We should not rely on the reporter to decide if a bug is performance-related. The model will be vulnerable to sudden quality regressions when people move around.